src/css/base.scss

```diff
@@ -114,33 +112,81 @@ h1 {
 img {
   max-width: 250px;
   height: auto;
-  // padding: 5%;
 }
 
 .right-side {
   grid-column: 2 / 4;
+  flex: 1 1;
   padding: 2rem;
-  background-color: v.$med-warm;
+  background-color: v.$med-cool;
   display: flex;
   justify-content: space-between;
+  overflow: auto;
 }
 
-.book-trip {
-  background-color: v.$light-warm;
+.plan-adventure {
+  flex: 1 1;
   display: flex;
-  padding: 2rem;
-  overflow: auto;
+  flex-direction: column;
+  align-items: center;
+  color: v.$dark;
+  font-size: larger;
+    button {
```

Comment:
I love this nesting right here, I didnt really mess with nesting at all so its interesting to see.